src/pages/NewSignin.vue

```diff
@@ -459,8 +463,8 @@ export default {
           );
           this.$router.push(`/form/${this.userInformation["student_id"]}`);
         } else {
-          if (this.$store.state.sessionData.session_id != null) {
-              // do not send logs for reports, gurukul
+          if (this.$store.state.sessionData.session_id != null && this.$store.state.authGroupData.name != "AFTesting") {
```

Comment:
Maybe have a global variable called "TESTING_ONLY" at the top of the file which is true only for `AFTesting` now, but can be expanded later. Will make it clearer what is going on to someone else who will read the code, and will also allow us to add different sort of testing etc with similar flow

Also add a comment when you define the global variable about the implications: (no ID checking, no data collection)